### ISSUE TITLE
Some variations missing in `default_matches`

### ISSUE THREAD

**Issue by suprMax** [emojione/emojione#526]
I'm parsing emoji.json to use in my app and after checking support against http://unicode.org/Public/emoji/4.0/emoji-test.txt file I noticed some variations are missing:

```
// 1f575-2640: woman detective
1f575-fe0f-200d-2640
1f575-200d-2640-fe0f
// 1f575-2642: man detective
1f575-fe0f-200d-2642
1f575-200d-2642-fe0f

// 1f3cc-2640: woman golfing
1f3cc-fe0f-200d-2640
1f3cc-200d-2640-fe0f
// 1f3cc-2642: man golfing
1f3cc-fe0f-200d-2642
1f3cc-200d-2642-fe0f

// 26f9-2640: woman bouncing ball
26f9-fe0f-200d-2640
26f9-200d-2640-fe0f
// 26f9-2642: man bouncing ball
26f9-fe0f-200d-2642
26f9-200d-2642-fe0f

// 1f3cb-2640: woman lifting weights
1f3cb-fe0f-200d-2640
1f3cb-200d-2640-fe0f
// 1f3cb-2642: man lifting weights
1f3cb-fe0f-200d-2642
1f3cb-200d-2642-fe0f

// 1f441-1f5e8: eye in speech bubble
1f441-fe0f-200d-1f5e8
1f441-200d-1f5e8-fe0f
```

Would it be possible to include them in JSON in the next version?

For everyone having the same issue here's how I monkey patch them right now:

```
const patchEmojioneSource = (object) => {
  const professions = [
    '1f575', // detective
    '1f3cc', // golfing
    '26f9', // bouncing ball
    '1f3cb', // lifting weights
  ];

  const female = '2640';
  const male = '2642';

  professions.forEach((prefix) => {
    [female, male].forEach((postfix) => {
      const array = object[`${prefix}-${postfix}`].code_points.default_matches;
      array.push(`${prefix}-fe0f-200d-${postfix}`);
      array.push(`${prefix}-200d-${postfix}-fe0f`);
    });
  });

  // eye in speech bubble
  const bubbleArray = object['1f441-1f5e8'].code_points.default_matches;
  bubbleArray.push('1f441-fe0f-200d-1f5e8');
  bubbleArray.push('1f441-200d-1f5e8-fe0f');

  // these are missing required fe0f ending and not rendered properly on iOS because of that
  // just swap with default matches that have that
  ['00a9', '00ae', '2122', '203c', '2049'].forEach((key) => {
    const item = object[key].code_points;
    const [full] = item.default_matches;
    item.default_matches = [key];
    item.output = full;
  });
};
```

Update: added emojis missing `fe0f` variation

**Comment by suprMax** [comment]
Also these emojis are missing the variation ending in the 
"output" in JSON:

`'00a9', '00ae', '2122', '203c', '2049'`

I noticed, that the JS library has correct "uc_output" though. @caseyahenson could you please share the generator script for the library, or update the JSON so the "output" field always has correct output sequence? Just sending what's in "output" right now doesn't show up as native emojis on iOS.

**Comment by caseyahenson** [comment]
@suprMax take a look at the updated emoji.json file. We included attributes for fully/non-fully qualified code points, plus the output you see there now includes vs16 for the code points you mentioned as well as others. Something we've dealt for awhile now has been the discrepancy between standardized output and matching for code points such as 00a9/00ae/2122/etc, and the format you might occasionally see in places like Emojipedia or the Unicode charts.  In some instances you'll see the NFQ variation used for particular emoji and we were sort of altering the standardized data to be able to match everything effectively. We improved our matching strategy by utilizing greedy matching in particular instances, but adding matching for NFQ code points was still necessary to catch everything. If you see anything that still doesn't line up right let us know.
